components/Router/Router.cjsx

```diff
@@ -19,6 +19,8 @@ ManageStepsExamples           = require '../ManageSteps/ManageStepsExamples.cjsx
 StepRowExamples               = require '../StepRow/StepRowExamples.cjsx'
 PanelExamples                 = require '../Panel/PanelExamples.jsx'
 StandardListItemExamples      = require '../StandardListItem/StandardListItemExamples.cjsx'
+NavbarDropdownExamples        = require '../NavbarDropdown/NavbarDropdownExamples.jsx'
```

Comment:
I think NavbarDropdownExamples should be removed from this PR because we are handling it with separate PR and it would create merge conflicts after we merge that other PR.